_ontimize_web_ngx_charts/configuration/00-overviewchart.md

```diff
@@ -0,0 +1,129 @@
+---
+permalink: /charts/configuration/overview/
+title: "Overview"
+comp: overviewchart
+layout: o-component
+---
+
+{% include base_path %}
+{% include toc %}
+
+## Introduction
+
+
+The configuration is used to change how the chart behaves. There are properties to control the type of chart displayed, styling, fonts, the legend, etc.
+You can use inputs and outputs to configure your charts and to catch events when are triggered.
+
+You can check every *chart parameter* [here](https://nvd3-community.github.io/nvd3/examples/documentation.html).
```

Comment:
it would be better to refer to the ontimize-web-chart api because all the options are not implemented